x-pack/plugins/ml/server/models/data_frame_analytics/index_patterns.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { APICaller, SavedObjectsClientContract } from 'kibana/server';
+import { IndexPatternAttributes } from 'src/plugins/data/server';
+// import { findObjectByTitle } from 'src/plugins/saved_objects/';
+import { mlLog } from '../../client/log';
+export const SAVED_OBJECT_TYPES = {
+  DASHBOARD: 'dashboard',
+  SEARCH: 'search',
+  VISUALIZATION: 'visualization',
+};
+
+export class IndexPatternHandler {
+  modulesDir = `${__dirname}/modules`;
+  indexPatternName: string = '';
+  indexPatternId: string | undefined = undefined;
+
+  constructor(
+    private callAsCurrentUser: APICaller,
```

Comment:
I don't think you are using `callAsCurrentUser`

Reply:
Removed in https://github.com/elastic/kibana/commit/d14fafab86d64ef3afce4bda0f3c415ea85e672e#diff-d71b0ca06f7c6aa048b8b3f9bb7c2eebR11